package.json

```diff
@@ -1,15 +1,19 @@
 {
   "private": true,
   "scripts": {
-    "build": "pulp build --to dist/app.js",
-    "watch": "pulp -w build --to dist/app.js",
-    "watch-fast": "webpack --mode=development --entry ./entry.js --output-path ./dist --output-filename app.js --progress --watch"
+    "postinstall": "spago install",
```

Comment:
I think we can skip this, since `spago` does this itself before commands that need it: 

https://github.com/purescript/spago/#download-my-dependencies-locally
> However, running this directly is usually not necessary, as all commands that need the dependencies to be installed will run this for you.

Reply:
We can skip it. I usually add this for projects that have separate test / main spago.dhall files, as then the postinstall would be `spago --config test.dhall install` to get the dev dependencies, but here we don't need it.